pkg/microservice/aslan/core/delivery/service/version.go

```diff
@@ -1113,13 +1106,26 @@ func sendVersionDeliveryHook(projectName, version, host, urlPath string) error {
 func updateVersionStatus(versionName, projectName, status, errStr string) {
```

Comment:
this function is mostly called by server-side logic, errors will only be logged instead of returned to client. And the error of sending request to hook url should not block the following db update operation, so errors are handlered(logger) inner the function